### PR TITLE
ADTS-80 Adding a support Disclaimer for China Conf

### PR DESCRIPTION
**Pull request by JoseZarmada**

### CODE REVIEW

File: client/src/components/ErrorMessageComponent/ErrorMessage.js [hunk 1]
```diff
@@ -80,7 +92,7 @@ export class ErrorMessageComponent extends Component {
   }
 
   render() {
-    const { showModal, errorMessage, stackErrorMessage, showFixAuth, showAuthSpinner, showAuthStatus, showAuthResponse} = this.state;
+    const { showModal, errorMessage, stackErrorMessage, showFixAuth, showAuthSpinner, showAuthStatus, showAuthResponse, appAdtChinaUrl} = this.state;
```
Comment (jamescarpinter) [inline client/src/components/ErrorMessageComponent/ErrorMessage.js]: Can we add a space between the Url and the } character?

Comment (JoseZarmada) [reply client/src/components/ErrorMessageComponent/ErrorMessage.js]: Ready

File: client/src/components/ErrorMessageComponent/ErrorMessage.js [hunk 2]
```diff
@@ -59,6 +64,13 @@ export class ErrorMessageComponent extends Component {
     this.setState({ showModal: false });
   }
 
+  checkUrl = async () => {
+    const { appAdtUrl } = await configService.getConfig();
+    const urlValues = appAdtUrl.split(".");
+    const chinaUrl = urlValues[urlValues.length - 1] === "cn";
```
Comment (jamescarpinter) [inline client/src/components/ErrorMessageComponent/ErrorMessage.js]: Can we add a check here for the length of the urlValues array? I can't imagine this is a common scenario but just to be on the safe side. I.e. `urlValues.length > 0 && urlValues[urlValues.length - 1] === "cn";`.

File: client/src/components/ErrorMessageComponent/ErrorMessage.js [hunk 3]
```diff
@@ -113,7 +125,10 @@ export class ErrorMessageComponent extends Component {
         isVisible={showModal}
         className="error-message">
         <div className="message-container">
-          <h2 tabIndex="0" className="heading-2" id="error-message-header"><span>!</span>Error</h2>
+          <h2 tabIndex="0" className="heading-2" id="error-message-header"><span>!</span>{this.props.t("errorMessages.error")}</h2>
+          { appAdtChinaUrl && <MessageBar>
```
Comment (jamescarpinter) [inline client/src/components/ErrorMessageComponent/ErrorMessage.js]: Remove the space between the bracket and variable name.

File: client/src/components/ErrorMessageComponent/ErrorMessage.js [hunk 4]
```diff
@@ -23,7 +25,8 @@ export class ErrorMessageComponent extends Component {
       showAuthSpinner: false,
       showAuthStatus: 0,
       showAuthResponse: false,
-      stackErrorMessage: ""
+      stackErrorMessage: "",
+      appAdtChinaUrl: false
```
Comment (jamescarpinter) [inline client/src/components/ErrorMessageComponent/ErrorMessage.js]: A better name might be `showAdtChinaWarningMessage` or similar?

File: client/src/components/ErrorMessageComponent/ErrorMessage.js [hunk 5]
```diff
@@ -2,13 +2,15 @@
...
 import React, { Component } from "react";
-import { DefaultButton, Spinner } from "office-ui-fabric-react";
+import { DefaultButton, Spinner, MessageBar } from "office-ui-fabric-react";
 import ModalComponent from "../ModalComponent/ModalComponent";
 import { eventService } from "../../services/EventService";
 import { CUSTOM_AUTH_ERROR_MESSAGE, MORE_INFORMATION_LINK, CUSTOM_NOT_FOUND_ERROR_MESSAGE, CUSTOM_AZURE_ERROR_MESSAGE, AUTH_SUCCESS_MESSAGE,
   AUTH_CONFLICT_MESSAGE, AUTH_FORBIDDEN_MESSAGE, AUTH_NOT_FOUND_MESSAGE } from "../../services/Constants";
 import { print } from "../../services/LoggingService";
 import { rbacService} from "../../services/RBACService";
+import { withTranslation } from "react-i18next";
```
Comment (jamescarpinter) [inline client/src/components/ErrorMessageComponent/ErrorMessage.js]: We _tend_ to put the npm imports above the file imports.

**Review by jamescarpinter — CHANGES_REQUESTED** [review]

File: client/src/components/ErrorMessageComponent/ErrorMessage.js (hunk 2 above)
Comment (JoseZarmada) [reply client/src/components/ErrorMessageComponent/ErrorMessage.js]: Ready

File: client/src/components/ErrorMessageComponent/ErrorMessage.js (hunk 3 above)
Comment (JoseZarmada) [reply client/src/components/ErrorMessageComponent/ErrorMessage.js]: Ready

File: client/src/components/ErrorMessageComponent/ErrorMessage.js (hunk 4 above)
Comment (JoseZarmada) [reply client/src/components/ErrorMessageComponent/ErrorMessage.js]: Ready

File: client/src/components/ErrorMessageComponent/ErrorMessage.js (hunk 5 above)
Comment (JoseZarmada) [reply client/src/components/ErrorMessageComponent/ErrorMessage.js]: Ready

**Review by jamescarpinter — APPROVED** [review]

**Pull request merged**